openquake/risklib/scientific.py

```diff
@@ -50,7 +50,13 @@
 structural+nonstructural_ins structural+contents_ins nonstructural+contents_ins
 structural+nonstructural+contents_ins
 structural_ins nonstructural_ins
-reinsurance claim area number'''.split())
+reinsurance claim area number
+structural+business_interruption nonstructural+business_interruption
+contents+business_interruption
+structural+nonstructural+business_interruption
+structural+contents+business_interruption
+nonstructural+contents+business_interruption
+structural+nonstructural+contents+business_interruption'''.split())
```

Comment:
We need to add exactly the same loss types but with an `_ins` suffix.

Reply:
Done